ui/imports/utils/Constants.qml

```diff
@@ -20,17 +20,6 @@ QtObject {
         readonly property int node: 7
     }
 
-    readonly property QtObject appViewStackIndex: QtObject {
```

Comment:
@alexandraB99 why did you remove this? This is very important and mandatory part in order to have everything in place correctly.